core/templates/dev/head/profile/profile.html

```diff
@@ -128,10 +128,11 @@ <h2 class="oppia-profile-username-small-screen">
       </md-content>
 
       <span class="oppia-profile-portfolio-pages" ng-if="numUserPortfolioExplorations > 6">
-        <li class="glyphicon glyphicon-arrow-left" ng-disabled="currentPageNumber == 0" ng-click="goToPreviousPage()"></li>
+        <li ng-disabled="currentPageNumber == 0" ng-click="goToPreviousPage()"><i class="material-icons">&#xE5C4;</i></li>
         Showing <[currentPageNumber * PAGE_SIZE + 1]> - <[Math.min(numUserPortfolioExplorations, (currentPageNumber + 1) * PAGE_SIZE)]> of <[numUserPortfolioExplorations]>
-        <li class="glyphicon glyphicon-arrow-right" ng-disabled="(currentPageNumber + 1) * PAGE_SIZE >= userEditedExplorations.length"
+        <li ng-disabled="(currentPageNumber + 1) * PAGE_SIZE >= userEditedExplorations.length"
             ng-click="goToNextPage()">
```

Comment:
Can you pull this up with line 133 ? And use proper indentation for <i> tag below.